src/main/java/aquality/appium/mobile/elements/actions/ElementTouchActions.java

```diff
@@ -0,0 +1,122 @@
+package aquality.appium.mobile.elements.actions;
+
+import aquality.appium.mobile.actions.SwipeDirection;
+import aquality.appium.mobile.actions.TouchActions;
+import aquality.appium.mobile.application.AqualityServices;
+import aquality.appium.mobile.configuration.ISwipeConfiguration;
+import aquality.appium.mobile.elements.interfaces.IElement;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.Point;
+
+public class ElementTouchActions implements IElementTouchActions {
+
+    private final Point scrollDownStartPoint = recalculatePointCoordinates(
+            getBottomRightCornerPoint(),
+            AqualityServices.get(ISwipeConfiguration.class).getHorizontalSwipeBottomPointXCoefficient(),
+            AqualityServices.get(ISwipeConfiguration.class).getHorizontalSwipeBottomPointYCoefficient());
+
+    private final Point scrollDownEndPoint = recalculatePointCoordinates(
+            getBottomRightCornerPoint(),
+            AqualityServices.get(ISwipeConfiguration.class).getHorizontalSwipeTopPointXCoefficient(),
+            AqualityServices.get(ISwipeConfiguration.class).getHorizontalSwipeTopPointYCoefficient());
+
+    private final Point swipeLeftStartPoint = recalculatePointCoordinates(
+            getBottomRightCornerPoint(),
+            AqualityServices.get(ISwipeConfiguration.class).getVerticalSwipeRightPointXCoefficient(),
+            AqualityServices.get(ISwipeConfiguration.class).getVerticalSwipeRightPointYCoefficient());
```

Comment:
we should probably have a single place where we get a ISwipeConfiguration in this class